### ISSUE TITLE
npm init minze@latest fails

### ISSUE THREAD

**Issue by rokyed** [n6ai/minze#168]
### Describe the bug

Node version 16 and 18.

``` bash
Error: ENOENT: no such file or directory, scandir '.../.npm/_npx/[id]/node_modules/create-minze/template-ts'
  at Object.readdirSync (node:fs:1451:3)
  at run (.../.npm/_npx/[id]/node_modules/create-minze/index.js:146:20) 
  ... {
errno: -2
syscall: 'scandir',
code: 'ENOENT',
path: '.../.npm/_npx/[id]/node_modules/create-minze/template-ts'
}
```

### Reproduction

run `npm init minze@latest ./my-project`

### System Info

```shell
System:
    OS: Linux 6.2 Pop!_OS 22.04 LTS
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
    Memory: 42.47 GB / 62.58 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.12.0 - ~/.nvm/versions/node/v18.12.0/bin/node
    npm: 9.1.1 - ~/.nvm/versions/node/v18.12.0/bin/npm
  Browsers:
    Brave Browser: 114.1.52.126
    Chrome: 114.0.5735.133
    Firefox: 114.0.1
```
```


### Used Package Manager

npm

### Logs

_No response_

### Validations

- [X] Follow our [Code of Conduct](https://github.com/n6ai/minze/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/n6ai/minze/blob/main/.github/CONTRIBUTING.md).
- [X] Read the [docs](https://minze.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/n6ai/minze/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/n6ai/minze/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by rokyed** [comment]
I forgot to mention but checked the folder and it's missing the template folders:
``` bash
cd ~/.npm/_npx/[id]/node_modules/create-minze
ls -la
drwxrwxr-x 2 user group 4096 Jun 16 10:35 .
drwxrwxr-x 9 user group 4096 Jun 16 10:35 ..
-rwxrwxr-x 1 user group 5974 Jun 16 10:35 index.js
-rw-rw-r-- 1 user group 1092 Jun 16 10:35 LICENSE
-rw-rw-r-- 1 user group  798 Jun 16 10:35 package.json
-rw-rw-r-- 1 user group  839 Jun 16 10:35 README.md
```

**Comment by sergejcodes** [comment]
Templates are now included in the latest release of create-minze `v1.2.1`. npm made some changes to its package publishing file/folder-includes system. Previously the templates were included via a wildcard syntax `template-*`, this isn't possible anymore for some reason.
